client-core/src/wallet/default_wallet_client.rs

```diff
@@ -90,10 +89,30 @@ where
     }
 
     fn new_wallet(&self, name: &str, passphrase: &SecUtf8) -> Result<()> {
-        let view_key = self.key_service.generate_keypair(passphrase)?.0;
+        println!("DefaultWalletClient New Wallet");
+        println!(
+            "is hd wallet={}",
+            self.key_service.is_hd_wallet(name, passphrase)
+        );
```

Comment:
Maybe use `debug!` instead, or remove them if possible?

Reply:
i will change to debug!